SystemInfo/Collector/ConfigurationSymfonyKernelSystemInfoCollector.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/**
+ * File containing the ConfigurationSymfonyKernelSystemInfoCollector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Collector;
+
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+use Symfony\Component\HttpKernel\Kernel;
+
+/**
+ * Collects information about the Symfony installation we are using.
+ */
+class ConfigurationSymfonyKernelSystemInfoCollector implements SystemInfoCollector
+{
+    /**
+     * Symfony environment.
+     *
+     * "dev" or "prod".
+     *
+     * @var string
+     */
+    private $environment;
+
+    /**
+     * True if Symfony is in debug mode.
+     *
+     * @var bool
+     */
+    private $debugMode;
+
+    /**
+     * An array containing the active bundles (keys) and the corresponding namespace.
+     *
+     * @var array
+     */
+    private $bundles;
+
+    /**
+     * Root directory.
+     *
+     * @var string
+     */
+    private $rootDir;
+
+    /**
+     * Name.
+     *
+     * @var string
+     */
+    private $name;
+
+    /**
+     * Cache directory.
+     *
+     * @var string
+     */
+    private $cacheDir;
+
+    /**
+     * Log file directory.
+     *
+     * @var string
+     */
+    private $logsDir;
+
+    /**
+     * Character set.
+     *
+     * @var string
+     */
+    private $charset;
+
+    /**
+     * Container class.
+     *
+     * @var string
+     */
+    private $containterClass;
```

Comment:
Right, though I am no longer collecting this data. It was the name of the class. Now, when I'm passing the whole kernel object to the collector, I get the container object instead, and I'd have to get the class name from that. Not hard, but it seems a rather obscure/pointless thing to deal with. Let me know if you want it back.